paper/paper.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'wdl2cwl: A Python tool for converting WDL workflow files to CWL workflows'
+tags:
+  - Python
+  - workflow
+  - workflow description language
+  - common workflow language
+  - Language
+authors:
+  - name: Dinithi Wickramaratne
+  - name: Dennis Chukwunta
+  - name: Michael R. Crusoe
+    orcid: 0000-0002-2961-9670
+  - name: Bruno P. Kinoshita
+    orcid: 0000-0001-8250-4074
+  - name: Peter Amstutz
+    orcid: 0000-0003-3566-7705
+date:  August 2022
+bibliography: paper.bib
+---
+
+# Summary
+
+Computational workflows are used in data analysis, enabling innovation and 
+decision-making in various fields of science. There are several competing workflow description languages, 
+WDL (Workflow description Language) and CWL (Common Workflow Languaage) are two examples
+of popularly used systems. WDL is a way to specify data processiing workflows with 
+a human-readable and writeable syntax. WDL makes it straightforward to define 
+complex analysis tasks, chain them together in workflows, and parallelize their execution [@wdl].
+CWL is a way to describe command line tools and connect 
+the together to create workflows. Because CWL is a specification and not a 
+specific piece of software, tools and workflows described using CWL are portable acroos
+a variety of platforms that support the CWL standard [@cwl].
+
+It can be hard learning and adapting to the various knowledge requirements and
+specifications of a workflow language. This complication makes it hard for analysts and scientists 
+share their work and collaborate efficiently with others who are use to working with different language.
+
+``wdl2cwl`` is a Python package for converting workflow files written in WDL to workflows in CWL. Python
+allows flexibility and the ability to import other Python packages. 
+The ``wdl2cwl`` package uses the ``miniwdl`` package [@miniwdl], an open-source, Python package, 
+developer toolkit and local runner for running wdl files. The ``wdl2cwl`` package was
+designed to provide a class-based and developer-friendly interface for extracting 
+the various sections of a wdl file, dynamically create the various objects
+needed to create a valid cwl file ( this is done using the ``cwl.utils`` package [@cwl_utils], 
```

Comment:
Ditto for cwl and CWL :point_up:

Reply:
Oh, and `cwl.utils` is the Python package. But the Python module is `cwl-utils`. When I'm talking with other engineers, I think sometimes we use it interchangeably? But in a paper like this one we probably want to use the right term.

You are not wrong that we use the `cwl.utils` package, but I think we must refer to Python modules. You can read more about modules & packages in these two links if you'd like: https://realpython.com/python-modules-packages/ & https://docs.python.org/3/tutorial/modules.html